std_msgs/Header header
sensor_msgs/Image back_img
sensor_msgs/Image[] deco_imgs
std_msgs/Float64[] deco_scales
